website/integrations/services/grafana/index.mdx

```diff
@@ -26,6 +26,72 @@ Create an application in authentik. Create an OAuth2/OpenID provider with the fo
 
 Note the Client ID and Client Secret values. Create an application, using the provider you've created above. Note the slug of the application you've created.
 
+
+## Terraform provider
+
+```hcl
+
+data "authentik_flow" "grafana-authentication-flow" {
+  slug = "default-authentication-flow"
+}
+
+data "authentik_flow" "grafana-authorization-flow" {
+  slug = "default-provider-authorization-implicit-consent"
+}
+
+data "authentik_scope_mapping" "oauth-grafana-email" {
+  name = "authentik default OAuth Mapping: OpenID 'email'"
+}
+
+data "authentik_scope_mapping" "oauth-grafana-profile" {
+  name = "authentik default OAuth Mapping: OpenID 'profile'"
+}
+
+data "authentik_scope_mapping" "oauth-grafana-openid" {
+  name = "authentik default OAuth Mapping: OpenID 'openid'"
+}
+
+resource "authentik_scope_mapping" "grafana-login" {
+  name       = "grafana-login"
+  scope_name = "login"
+  expression = "return request.user.username"
+}
+
+resource "authentik_scope_mapping" "grafana-name" {
+  name       = "grafana-name"
+  scope_name = "name"
+  expression = "return request.user.name"
+}
+
+resource "authentik_provider_oauth2" "provider_grafana" {
+  name          = "Grafana"
+  client_id     = "my_client_id"
+  client_secret = "my_client_secret"
+
+  authentication_flow = data.authentik_flow.grafana-authentication-flow.id
+  authorization_flow  = data.authentik_flow.grafana-authorization-flow.id
+
+  redirect_uris = ["https://grafana.company/login/generic_oauth"]
+
+  property_mappings = [
+    authentik_scope_mapping.grafana-login.id,
+    authentik_scope_mapping.grafana-name.id,
+    data.authentik_scope_mapping.oauth-gitea-email.id,
+    data.authentik_scope_mapping.oauth-gitea-profile.id,
+    data.authentik_scope_mapping.oauth-gitea-openid.id,
+  ]
+}
+
+resource "authentik_application" "application_grafana" {
+  name              = "Grafana"
+  slug              = "grafana-slug"
+  protocol_provider = authentik_provider_oauth2.provider_grafana.id
+
+  meta_launch_url = "https://grafana.company/"
+}
```

Comment:
```suggestion
data "authentik_certificate_key_pair" "default" {
  name = "authentik Self-signed Certificate"
}

data "authentik_flow" "default-provider-authorization-implicit-consent" {
  slug = "default-provider-authorization-implicit-consent"
}

data "authentik_scope_mapping" "scope-email" {
  name = "authentik default OAuth Mapping: OpenID 'email'"
}

data "authentik_scope_mapping" "scope-profile" {
  name = "authentik default OAuth Mapping: OpenID 'profile'"
}

data "authentik_scope_mapping" "scope-openid" {
  name = "authentik default OAuth Mapping: OpenID 'openid'"
}

resource "authentik_provider_oauth2" "grafana" {
  name          = "Grafana"
  # Optional: will be generated if not provided
  # client_id     = "my_client_id"
  # client_secret = "my_client_secret"

  authorization_flow  = data.authentik_flow.default-provider-authorization-implicit-consent.id

  redirect_uris = ["https://grafana.company/login/generic_oauth"]

  property_mappings = [
    data.authentik_scope_mapping.scope-email.id,
    data.authentik_scope_mapping.scope-profile.id,
    data.authentik_scope_mapping.scope-openid.id,
  ]
  
    signing_key = data.authentik_certificate_key_pair.default.id
}

resource "authentik_application" "grafana" {
  name              = "Grafana"
  slug              = "grafana"
  protocol_provider = authentik_provider_oauth2.grafana.id
}
```

Updated the terraform to only have what's needed and to match with the other instructions.